mypy/semanal.py

```diff
@@ -2696,7 +2696,8 @@ def visit_member_expr(self, expr: MemberExpr) -> None:
                 # one type checker run. If we reported errors here,
                 # the build would terminate after semantic analysis
                 # and we wouldn't be able to report any type errors.
-                full_name = '%s.%s' % (file.fullname() if file is not None else None, expr.name)
+                full_name = '%s.%s' % (file.fullname() if file is not None
+                                       else None, expr.name)
```

Comment:
I think you don't need this formatting change, it only creates noise in the PR. Otherwise now it looks good to me